guides/source/active_job_basics.md

```diff
@@ -708,3 +894,58 @@ Debugging
 ---------
 
 If you need help figuring out where jobs are coming from, you can enable [verbose logging](debugging_rails_applications.html#verbose-enqueue-logs).
+
+Alternate Queuing Backends
+--------------------------
+
+Active Job has other built-in adapters for multiple queuing backends (Sidekiq,
+Resque, Delayed Job, and others). To get an up-to-date list of the adapters
+see the API Documentation for [`ActiveJob::QueueAdapters`][].
```

Comment:
[We're deprecating other queue adapters](https://github.com/rails/rails/issues/52929). They will live in their respective gems in the future. Should we rephrase this?

Reply:
Thanks for the headsup @MatheusRich! Is there an existing protocol for the Rails docs - do we mention the intention to deprecate in the future or only mention it once it has been deprecated?

cc @AmandaPerino @p8